app/models/user.rb

```diff
@@ -56,7 +56,7 @@ class User < ApplicationRecord
     where(casa_org_id: org.id)
   }
 
-  scope :no_recent_sign_in, -> { active.where("last_sign_in_at <= ?", 30.days.ago) }
+  scope :no_recent_sign_in, -> { active.where("last_sign_in_at <= ?", 30.days.ago).or(User.active.where(last_sign_in_at: nil)) }
```

Comment:
Added the or clause for users who had never signed in. Found the use case when viewing in the mailer preview.